products/jira.md

```diff
@@ -23,8 +23,8 @@ releases:
 -   releaseCycle: "9.10"
     releaseDate: 2023-06-11
     eol: 2025-07-11
-    latest: "9.10.1"
-    latestReleaseDate: 2023-08-03
+    latest: "9.10.2"
+    latestReleaseDate: 2023-09-19
```

Comment:
```suggestion
    latestReleaseDate: 2023-09-20
```

According to https://confluence.atlassian.com/jirasoftware/issues-resolved-in-9-10-2-1296171340.html.